src/dfx/src/actors/replica.rs

```diff
@@ -254,6 +254,9 @@ fn replica_start_thread(
             // For our production network, we actually set them to 600ms.
             "600",
         ]);
+
+        cmd.env("RUST_MIN_STACK", "8192000");
```

Comment:
Should we add a comment that this should agree with the value in https://gitlab.com/dfinity-lab/core/ic/-/blob/master/ic-os/guestos/rootfs/etc/systemd/system/ic-replica.service ?

Reply:
Yes, will do